### ISSUE TITLE
WARNING keyword

### ISSUE THREAD

**Issue by FerreolS** [JMMC-OpenDev/OI-Imaging-JRA#16]

**Comment by GillesDuvert** [comment]
Indeed, looking at the log is something people do only in the most dire circumstances...
The problem with the WARNING kw/value mechanism is that the value may be veeery long, e.g. a warning issued at each loop.  
And it adds yet another demand to the softwares, that will not be correctly fulfilled until long, and is against our  policy of adding more softwares without much trouble for their authors.
I suggest the GUI scans the log for a keyword (say, "WARNING" uppercase) and inform visually that there are wanrning(s) in the log. Which will need no change for Wisard as this is already done :smile:

**Comment by FerreolS** [comment]
@buthanoid ? Do you think that parsing the log within OImaging is something you can implement?

**Comment by buthanoid** [comment]
Yes

**Comment by emmt** [comment]
We can add more conventions for tagging the log messages so that they can be automatically reported in the gui.
